__tests__/api.test.js

```diff
@@ -442,3 +442,33 @@ describe("/api/users", () => {
       });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  test('DELETE: 204. Deletes comment and returns no content', () => {
+    return request(app)
+    .delete("/api/comments/1")
+    .expect(204)
+    .then(() => {
+      return db.query("SELECT * FROM comments WHERE comment_id = 1")
+    })
+    .then((result) => [
```

Comment:
Are the square brackets instead of curly braces deliberate here? I'll be honest in that I'm not sure I've seen this before or why the test is passing -  it seems it's valid syntax. You may be getting a false positive but you maybe doing something I've not seen before 🤯 😄

Reply:
This was an error (should be curly braces). I believe this technically is correct as we have implicit return of an array with a single element found in line 155, but this isn't sensible here so I've amended it